README.md

```diff
@@ -1,40 +1,44 @@
 [![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg?style=flat-square)](https://gitter.im/fossasia/open-event-frontend)
 [![Codacy grade](https://img.shields.io/codacy/grade/0d51cf60fc734d3699fd6eff6054e483.svg?style=flat-square)](https://www.codacy.com/app/niranjan94/open-event-frontend)
 [![codecov](https://img.shields.io/codecov/c/github/fossasia/open-event-frontend/development.svg?style=flat-square)](https://codecov.io/gh/fossasia/open-event-frontend)
-
 [![Greenkeeper badge](https://badges.greenkeeper.io/fossasia/open-event-frontend.svg)](https://greenkeeper.io/)
 [![Known Vulnerabilities](https://snyk.io/test/github/fossasia/open-event-frontend/badge.svg)](https://snyk.io/test/github/fossasia/open-event-frontend)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/open-event-frontend/localized.svg)](http://translate.eventyay.com/project/open-event-frontend)
 
-> The front end for the Open Event API Server
+> The front end for the **Open Event Server**
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
 **API Documentation:**
-- Every installation of the Open Event API Server project includes API docs, (e.g. here on the test install [https://open-event-api.herokuapp.com](https://open-event-api.herokuapp.com)).
-- A hosted version of the API docs is available in the `gh-pages` branch of the Open Event API Server repository at [http://dev.eventyay.com/api/v1](http://dev.eventyay.com/api/v1)
+- Every installation of the **Open Event Server** project includes API docs, (e.g. here on the test install [https://open-event-api.herokuapp.com](https://open-event-api.herokuapp.com)).
+- A hosted version of the API docs is available in the `gh-pages` branch of the **Open Event Server** repository at [http://dev.eventyay.com/api/v1](http://dev.eventyay.com/api/v1)
 
 ## Communication
 
-Please join our mailing list to discuss questions regarding the project: https://groups.google.com/forum/#!forum/open-event
+Please join our mailing list to discuss questions regarding the project :
+ 
+> *https://groups.google.com/forum/#!forum/open-event*
+
+Our chat channel is on Gitter here :
 
-Our chat channel is on Gitter here: [gitter.im/fossasia/open-event-frontend](https://gitter.im/fossasia/open-event-frontend)
+> *[gitter.im/fossasia/open-event-frontend](https://gitter.im/fossasia/open-event-frontend)*
 
 ## Prerequisites
 
 You will need the following things properly installed on your computer.
 
-* [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/) (with NPM v5.2x)
-* [Bower](https://bower.io/) - `npm install -g bower`
-* [Ember CLI](https://ember-cli.com/) - `npm install -g ember-cli`
-* [PhantomJS](http://phantomjs.org/) - `npm install -g phantomjs-prebuilt`
-* [check-node-version](https://www.npmjs.com/package/check-node-version) - `npm install -g check-node-version`
+* **[Git](https://git-scm.com/)**
```

Comment:
Please don't make these bold it starts to look cumbersome  :) 
![image](https://user-images.githubusercontent.com/17252805/33282001-35898ac2-d3cd-11e7-9481-7f0442646d8a.png)

Reply:
But, these changes were approved by other two repo's :open_mouth:

Reply:
@CosmicCoder96 You checked those repos ?